client/src/components/Footer.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Dice from '../icons/Dice';
+import AddQuestion from '../icons/AddQuestion';
+
+const FooterBar = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100vw;
```

Comment:
Do you need `vw` if your container works with `vh`/`vw`?

Reply:
probably would 100% work too...